src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/mappers/ChildrenInfoMapper.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.service.bulkscan.transformation.mappers;
+
+import org.apache.commons.lang3.StringUtils;
+import uk.gov.hmcts.reform.bsp.common.model.shared.in.OcrDataField;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.ChildInfo;
+import uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.ChildrenInfo;
+
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.hmcts.reform.bsp.common.mapper.GenericMapper.getValueFromOcrDataFields;
+import static uk.gov.hmcts.reform.bsp.common.utils.BulkScanCommonHelper.transformFormDateIntoCcdDate;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.service.bulkscan.transformation.mappers.ChildrenInfoMapper.Fields.COUNTRY;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.service.bulkscan.transformation.mappers.ChildrenInfoMapper.Fields.DOB;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.service.bulkscan.transformation.mappers.ChildrenInfoMapper.Fields.GENDER;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.service.bulkscan.transformation.mappers.ChildrenInfoMapper.Fields.NAME;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.service.bulkscan.transformation.mappers.ChildrenInfoMapper.Fields.RELATION_TO_APPLICANT;
+import static uk.gov.hmcts.reform.finrem.caseorchestration.service.bulkscan.transformation.mappers.ChildrenInfoMapper.Fields.RELATION_TO_RESPONDENT;
+
+public class ChildrenInfoMapper {
+
+    public static class Fields {
+        public static final String NAME = "NameOfChild";
+        public static final String GENDER = "GenderChild";
+        public static final String DOB = "DateOfBirthChild";
+        public static final String RELATION_TO_APPLICANT = "RelationshipToApplicantChild";
+        public static final String RELATION_TO_RESPONDENT = "RelationshipToRespondentChild";
+        public static final String COUNTRY = "CountryOfResidenceChild";
+    }
+
+    private ChildrenInfoMapper() {
+        // don't
+    }
```

Comment:
Remove this please

Reply:
Sonar gives us a code smell for leaving class (even with static methods only) without private constructor.

Reply:
![sonar](https://user-images.githubusercontent.com/30896346/79579083-05a48b80-80bf-11ea-803b-a51b877e7134.gif)
Is this something we have control over to edit so it doesnt? Having needless code included without any functionality is very unclean

Reply:
It does have functionality, prevents from instantiating the class. Could use lombok annotation instead, either way is fine.

Reply:
https://github.com/hmcts/finrem-case-orchestration-service/pull/293